examples/nginx-centos7/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+#
+# S2I save-artifacts script for the 'nginx-centos7' image.
+# The save-artifacts script streams a tar archive to standard output.
+# The archive contains the files and folders you want to re-use in the next build.
+#
+# For more information see the documentation:
+#	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+tar cf - /usr/share/nginx/html/
```

Comment:
I considered making a similar comment but i'm not actually sure what s2i will do w/ a save-artifacts script that doesn't output a tar stream to stdout, which is what would happen w/ your suggestion.  I suspect not good things.